src/utils/utils.js

```diff
@@ -900,6 +903,26 @@ export function getInSelection(document, selector) {
     );
 }
 
+/**
+ * Returns an array of url infos for url matched in the given string.
+ *
+ * @param {String} string
+ * @returns {Array}
+ */
+export function getUrlsInfosInString(string) {
+    let infos = [],
+        match;
+    while ((match = URL_REGEX_WITH_INFOS.exec(string))) {
+        infos.push({
+            url: match[2] ? match[0] : 'https://' + match[0],
+            label: match[0],
+            index: match.index,
+            length: match[0].length,
+        });
```

Comment:
Right so this would be the infamous `data` argument. In that case maybe a simple reference to this function in the docstring would be enough, though `data` and this differ in that `data` doesn't require `label` as the method is already getting a text node.